apacheconfig/loader.py

```diff
@@ -11,14 +11,16 @@
 import tempfile
 
 from apacheconfig.error import ApacheConfigError
+from apacheconfig.host import LocalHost
 
 log = logging.getLogger(__name__)
 
 
 class ApacheConfigLoader(object):
 
-    def __init__(self, parser, debug=False, **options):
+    def __init__(self, parser, host=LocalHost(), debug=False, **options):
```

Comment:
Given that there may be other sources of config files (other types of readers), may be kwarg like `reader` would better reflect the purpose of this parameter...? WDYT?

Reply:
I'll look into that, I'm knew to python and not really familiar with the way default, and optional parameters are handled here. But I like the idea of a kwarg. Also think passing just the class, like you suggested in your other comment, is a better way of handling the instantiation of the ```reader``` and will implement this during this step.